cmd/fleetctl/flags.go

```diff
@@ -44,3 +45,15 @@ func fleetCertificateFlag() cli.Flag {
 func getFleetCertificate(c *cli.Context) string {
 	return c.String(fleetCertificateFlagName)
 }
+
+func stdoutFlag() cli.Flag {
+	return &cli.BoolFlag{
+		Name:    stdoutFlagName,
+		EnvVars: []string{"STDOUT"},
+		Usage:   "Print contents to stdout",
+	}
+}
+
+func getStdout(c *cli.Context) bool {
+	return c.Bool(stdoutFlagName)
+}
```

Comment:
IMO for these one-liners it's more clear to just call `c.Bool(stdoutFlagName)` inlined, (or maybe just rename the method to `getStdoutFlag`, I was confused at first sight).

Reply:
I agree, I decided to go with this to be consistent with `getDebug` in this file. Maybe it's worth changing both?

Reply:
Just a nit, changes look good!

Reply:
gotcha, thank you! I will follow up in a PR (good to know I wasn't the only confused with the naming)